server/text_generation_server/layers/marlin/fp8.py

```diff
@@ -39,7 +39,8 @@ def __init__(
         log_once(logger.info, "GPU does not support FP8, using Marlin FP8 kernel")
 
         scales = scales.unsqueeze(0)
-        if scales.shape[1] == 1:
+        # repack weights for Marlin if a single scale is provided
+        if scales.size(0) == 1:
```

Comment:
```suggestion
        if scales.shape[0] == 1:
```

Can you explain where this change is coming from ?

Reply:
yes, apologies for not including an example above. Currently if attempting to quantize an unquantized model with marlin fp8 the line above will throw when attempting to use marlin to repack.

```
text-generation-launcher --model-id meta-llama/Meta-Llama-3-8B --quantize fp8
```

with the change above this model loads and generates as expected

```bash
curl 127.0.0.1:3000/generate -X POST -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":20}}' -H 'Content-Type: application/json'
# {"generated_text":" Deep learning is a subset of machine learning that is inspired by the structure and function of the human brain"}
```